x-pack/plugins/security_solution/server/lib/detection_engine/routes/signals/set_alert_tags_route.ts

```diff
@@ -90,9 +90,20 @@ export const setAlertTagsRoute = (router: SecuritySolutionPluginRouter) => {
       }
 
       try {
-        const body = await esClient.bulk({
-          refresh: 'wait_for',
-          body: bulkUpdateRequest,
+        const body = await esClient.updateByQuery({
+          index: `${DEFAULT_ALERTS_INDEX}-${spaceId}`,
+          // https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-update-by-query.html#_refreshing_shards_2
+          // Note: Before we tried to use "refresh: wait_for" but I do not think that was available and instead it defaulted to "refresh: true"
+          // but the tests do not pass with "refresh: false". If at some point a "refresh: wait_for" is implemented, we should use that instead.
+          refresh: true,
```

Comment:
same as above

> is allowing refresh:true something we want to do?
I remember we did some work to avoid explicitly refreshing indices: https://github.com/elastic/security-team/issues/6561